src/components/molecules/DatePickerInput/DatePickerInput.tsx

```diff
@@ -24,13 +25,76 @@ export interface DatePickerInputProps {
   placeholder?: string
   disabled?: boolean
   value?: string
+  timePicker?: boolean
+  onChange: (value: string) => void
+  showSeconds?: boolean
+}
+
+export interface DatePickerComponentProps {
+  ariaLabelToUse?: string
+  value?: string
+  placeholder?: string
+  disabled?: boolean
+  ariaLabel: string
+  timePicker?: boolean
   onChange: (value: string) => void
 }
 
 registerLocale('et-EE', et)
 
-const changeDateToString = (dateObject: Date | null | undefined) =>
-  dayjs(dateObject).format('DD/MM/YYYY')
+const changeDateToString = (dateObject: Date | null | undefined) => {
+  return dayjs(dateObject).format('DD/MM/YYYY')
+}
+
+const DatePickerComponent = ({
+  value,
+  placeholder,
+  ariaLabelToUse,
+  disabled,
+  ariaLabel,
+  timePicker,
+  onChange,
+  ...rest
+}: DatePickerComponentProps) => {
+  const handleDateChange: ReactDatePickerProps['onChange'] = (value) => {
+    return onChange(changeDateToString(value))
+  }
+
+  const splittedDayValue = value?.split('/')
+
+  const formattedDayValue =
+    splittedDayValue?.[2] +
+    '-' +
+    splittedDayValue?.[1] +
+    '-' +
+    splittedDayValue?.[0]
+
+  if (timePicker) return null
+
+  return (
+    <>
+      <DatePicker
+        id="DatePicker"
+        selected={value ? new Date(formattedDayValue) : null}
+        dateFormat={'dd.MM.yyyy'}
+        locale="et-EE"
+        placeholderText={placeholder}
+        aria-label={ariaLabelToUse || ''}
+        disabled={disabled}
+        {...rest}
+        onChange={handleDateChange}
+      />
+      <Icon
+        icon={Calender}
+        className={classNames(
+          classes.dateIcon,
+          disabled && classes.disabledCalender
+        )}
+        ariaLabel={ariaLabel}
```

Comment:
**[blocker]** icons don't need aria labels. aria-label is meant for interactive elements: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label